pytorch_lightning/accelerators/accelerator.py

```diff
@@ -24,6 +24,11 @@
 from pytorch_lightning.utilities.distributed import all_gather_ddp_if_available
 from pytorch_lightning.utilities.enums import AMPType, LightningEnum
 
+if TYPE_CHECKING:
+    from torch.cuda.amp import GradScaler
```

Comment:
this can be imported normally, right?

Reply:
It could, but it is not needed during runtime, so why import it then?